microsite/data/plugins/dev-quotes.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: Dev Quotes
+author: Peter Macdonald
+authorUrl: https://github.com/Parsifal-M
+category: Humor
+description: Displays a coding/progamming related quote designed as a footer for the Homepage, although to be honest it can be used anywhere you like!
+documentation: https://github.com/Parsifal-M/backstage-dev-quotes?tab=readme-ov-file#dev-quotes-homepage
+iconUrl: /img/dqicon.png
+npmPackageName: '@parsifal-m/plugin-dev-quotes-homepage'
```

Comment:
Could you add a link back to your git repo from the NPM package? Just adds that extra layer of trust

Reply:
Absolutely! :+1: